stream-chat-android-client/src/main/java/io/getstream/chat/android/client/events/ChatEvent.kt

```diff
@@ -23,348 +22,446 @@ public interface UserEvent {
     public val user: User
 }
 
-public data class ChannelCreatedEvent(
-    override val type: String,
-    @SerializedName("created_at") override val createdAt: Date,
-    override val cid: String,
-    @SerializedName("channel_type") val channelType: String,
-    @SerializedName("channel_id") val channelId: String,
-    override val user: User,
-    val message: Message?,
-    val channel: Channel,
-) : CidEvent(), UserEvent
+public interface HasChannel {
```

Comment:
We had `CidEvent` interface for this kind of events

Reply:
Moved `channelType` and `channelId` to `CidEvent`:
```
public sealed class CidEvent : ChatEvent() {
    public abstract val cid: String
    public abstract val channelType: String
    public abstract val channelId: String
}
``` 
`HasChannel` interface now contains only full `Channel` object